pkg/bff/api/v1/api.go

```diff
@@ -389,3 +390,17 @@ type NetworkError struct {
 	TestNet string `json:"testnet,omitempty"`
 	MainNet string `json:"mainnet,omitempty"`
 }
+
+// Activity is a time-aggregated collection of events (Search, Lookup, etc)
+type Activity struct {
+	Date   string `json:"date"`
+	Events int    `json:"events"`
+}
+
+// NetworkActivityReply is a map of the network (TestNet or MainNet) to Activity,
+// which is a time-aggregated collection of events (Search, Lookup, etc)
+// that occurred on the network
+type NetworkActivityReply struct {
+	TestNet []Activity `json:"testnet"`
+	MainNet []Activity `json:"mainnet"`
```

Comment:
Cool, I like how this is broken down and should be compatible with what @daniellemaxwell was planning.